Makefile

```diff
@@ -87,7 +91,7 @@ pyclean:
 
 .PHONY: envclean
 envclean:
-	rm -rf bin .eggs .develop-eggs parts .installed.cfg bootstrap.py .downloads .buildout_downloads
+	rm -rf bin .wheelhouse .wheels_installed .develop-eggs parts .installed.cfg bootstrap.py .downloads .buildout_downloads
```

Comment:
What command cleans up `${WHEELHOUSE}`?

Reply:
Ah, it doesn't exist. Cool

Reply:
To be clear, `${WHEELHOUSE}` belongs to the calling package, so it's the calling packages responsibility for cleaning it up.